files.go

```diff
@@ -145,6 +146,44 @@ type ListFilesParameters struct {
 	Cursor  string
 }
 
+type FileUploadV2Parameters struct {
```

Comment:
[IMO] Prefer `UploadFileV2Parameters`. It would be easier to use if aligned with the method name.